src/styles/main.scss

```diff
@@ -0,0 +1,49 @@
+@import "color";
+
+:is(main) {
```

Comment:
```
warnings when minifying css:
▲ [WARNING] A nested style rule cannot start with "main" because it looks like the start of a declaration [css-syntax-error]

    <stdin>:722:2:
      722 │   main {
          │   ~~~~
          ╵   :is(main)

  To start a nested style rule with an identifier, you need to wrap the identifier in ":is(...)" to prevent the rule from being parsed as a declaration.
```